src/rtcModule/webrtcPrivate.h

```diff
@@ -64,36 +98,45 @@ class Session: public ISession
     unsigned int mPreviousStatsSize = 0;
     std::unique_ptr<AudioLevelMonitor> mAudioLevelMonitor;
     TermCode mTermCode = TermCode::kInvalid;
+    bool mPeerSupportRenegotiation = false;
+    bool mRenegotiationInProgress = false;
```

Comment:
I wonder if this flag should be a new state of the session/call, rather than a state's flag. Let's discuss about it.

Reply:
It's not really necessary, session is terminating when the flag is set